### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: wellregistry/wellregistry/asgi.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for wellregistry project.
```
Comment (mbucknell) [inline wellregistry/wellregistry/asgi.py:2]: Are we going to be using asgi?

File: wellregistry/wellregistry/settings.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__)))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('SECRET_KEY')
```
Comment (mbucknell) [inline wellregistry/wellregistry/settings.py:24]: If we deploy from EROS, I'm guessing we will use Ansible in which cause this won't be retrieved from the environment.

File: wellregistry/wellregistry/asgi.py (hunk 1 above)
Comment (ayan-usgs) [reply wellregistry/wellregistry/asgi.py:2]: Nope. I'll yank it out.

File: wellregistry/wellregistry/settings.py (hunk 2 above)
Comment (ayan-usgs) [reply wellregistry/wellregistry/settings.py]: Added an import from `local_settings.py`.

**Pull request merged**